cdap-docs/admin-manual/source/index.rst

```diff
@@ -101,13 +101,6 @@ installation and its security configuration.
 - |verification|_ How to verify the CDAP installation on your Hadoop cluster by using an
   **example application** and **health checks**.
 
-
```

Comment:
are we going to revert this PR when we do 6.1.0 launch?

Reply:
Yes. https://issues.cask.co/browse/CDAP-15401